src/qt/configurenamedialog.cpp

```diff
@@ -51,6 +53,7 @@ void ConfigureNameDialog::accept()
         return;
 
     QString addr = ui->transferTo->text();
+    std::string data = ui->dataEdit->text().toStdString();
```

Comment:
Is there a reason not to use `returnData`? (Maybe there is, it just jumps out at me as a plausible alternative.)

Reply:
I did that and it doesn't work anymore, no idea why

Reply:
By it I mean the dialogue box, again no idea why

Reply:
You did run `.toStdString()` on `returnData` when trying it, right? If that doesn't work, then I won't worry about why, just was curious.